commit fb47e5911366daba6e29c5909880e3ca99ea4d90
Author: Jan Engelhardt <jengelh@inai.de>
Date:   Sun Jun 30 10:14:21 2019 +0200

Add some distinguishing library name identifiers as per
https://en.opensuse.org/openSUSE:Shared_library_packaging_policy#When_there_is_no_versioning
ordinal clause 3 (packaging Git snapshot with ABI changing midway).

---
 CMakeLists.txt |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

Index: vmime-0.9.2.165/CMakeLists.txt
===================================================================
--- vmime-0.9.2.165.orig/CMakeLists.txt
+++ vmime-0.9.2.165/CMakeLists.txt
@@ -50,7 +50,7 @@ SET(VMIME_VERSION_MICRO  2)
 # . Interfaces added/removed/changed:               CURRENT++, REVISION=0
 # . Interfaces added (upward-compatible changes):   AGE++
 # . Interfaces removed:                             AGE=0
-SET(VMIME_API_VERSION_CURRENT   1)
+SET(VMIME_API_VERSION_CURRENT   4)
 SET(VMIME_API_VERSION_REVISION  0)
 SET(VMIME_API_VERSION_AGE       0)
 
@@ -163,11 +163,15 @@ IF(VMIME_BUILD_SHARED_LIBRARY)
 	SET_TARGET_PROPERTIES(
 		${VMIME_LIBRARY_NAME}
 		PROPERTIES
+		OUTPUT_NAME "vmime-suse"
 		VERSION    "${VMIME_API_VERSION}"
 		SOVERSION  "${VMIME_API_VERSION_CURRENT}"
 		COMPILE_FLAGS -DVMIME_SHARED
 	)
 
+	execute_process(COMMAND ln -s libvmime-suse.so.${VMIME_API_VERSION_CURRENT} libvmime.so)
+	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libvmime.so DESTINATION ${CMAKE_INSTALL_LIBDIR}/ COMPONENT libraries)
+
 ENDIF()
 
 # Static library
